src/client/pages/OfferDebugger/components/QuoteData.tsx

```diff
@@ -0,0 +1,393 @@
+    case QuoteBundleType.SwedishApartment:
+      return QuoteType.SwedishApartment
+    case QuoteBundleType.SwedishHouse:
+      return QuoteType.SwedishHouse
+    default:
+      throw new Error(`Unsupported single quote bundle type: ${bundleType}`)
+  }
+}
+
+type QuoteData = {
+  label: string
+  value: QuoteBundleType
+  initialFormValues?: Record<string, unknown>
+}
+
+type QuotesByMarket = Record<Market, QuoteData[]>
+
+const quotesByMarket: QuotesByMarket = {
+  DK: [
+    {
+      label: 'Danish Home',
+      value: QuoteBundleType.DanishHome,
+      initialFormValues: initialDkHomeValues,
+    },
+    {
+      label: 'Danish Home + Accident',
+      value: QuoteBundleType.DanishHomeAccident,
+      initialFormValues: initialDkHomeValues,
+    },
+    {
+      label: 'Danish Home + Accident + Travel',
+      value: QuoteBundleType.DanishHomeAccidentTravel,
+      initialFormValues: initialDkHomeValues,
+    },
+  ],
+  NO: [
+    {
+      label: 'Norwegian Home',
+      value: QuoteBundleType.NorwegianHome,
+      initialFormValues: initialNoHomeValues,
+    },
+    {
+      label: 'Norwegian Travel',
+      value: QuoteBundleType.NorwegianTravel,
+      initialFormValues: initialNoTravelValues,
+    },
+  ],
+  SE: [
+    {
+      label: 'Swedish Apartment',
+      value: QuoteBundleType.SwedishApartment,
+      initialFormValues: initialSeApartmentValues,
+    },
+    {
+      label: 'Swedish House',
+      value: QuoteBundleType.SwedishHouse,
+    },
+    {
+      label: 'Swedish Apartment + Accident',
+      value: QuoteBundleType.SwedishApartmentAccident,
+      initialFormValues: initialSeApartmentValues,
+    },
+  ],
+}
+
+const getCurrentAvailableQuoteData = (
+  currentMarket: Market,
+  currentQuoteType: QuoteBundleType,
+) => {
+  const marketQuoteTypes = quotesByMarket[currentMarket]
+  const currentQuoteTypeData = marketQuoteTypes.find(
+    ({ value }) => value === currentQuoteType,
+  )
+  return currentQuoteTypeData
+}
+
+const getDanishQuoteValues = (
+  values: any,
+  quoteType: 'danishTravelData' | 'danishAccidentData',
+) => {
+  const { danishHomeContentsData, ...filteredValues } = values
+  const { subType, livingSpace, ...quoteTypeValues } = danishHomeContentsData!
+  return {
+    ...filteredValues,
+    [quoteType]: quoteTypeValues,
+  }
+}
+
+const getSwedishAccidentQuoteValues = (values: any) => {
+  const { swedishApartmentData, ...filteredValues } = values
+  const { subType, ...quoteTypeValues } = swedishApartmentData!
+  return {
+    ...filteredValues,
+    swedishAccidentData: {
+      ...quoteTypeValues,
+      student:
+        subType === ApartmentType.StudentBrf ||
+        subType === ApartmentType.StudentRent,
+    },
+  }
+}
+
+const ButtonLoadingIndicator = styled(LoadingDots)`
+  display: inline-flex;
+  margin-left: 0.5rem;
+`
+
+export const QuoteData: React.FC<OfferProps> = ({ quoteCartId }) => {
+  const currentLocale = useCurrentLocale()
+  const currentMarket = useMarket()
```

Comment:
Can we instead use `useCurrentLocale` from `/l10n/useCurrentLocale.ts` and then pick the `market` property from  the returned `locale` object instead? 🙏 I'm trying to get rid of all these old locale/market utils functions (and yes, I should have marked them as deprecated but I didn't think about that 🙈)